packages/checkbox-group/test/checkbox-group.test.js

```diff
@@ -2,6 +2,7 @@ import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
 import { fixtureSync, focusin, focusout } from '@vaadin/testing-helpers';
 import '@polymer/polymer/lib/elements/dom-bind.js';
+import '@vaadin/checkbox/src/vaadin-checkbox.js';
```

Comment:
This import shouldn't be needed here, as the component itself imports `vaadin-checkbox`?

Reply:
`vaadin-checkbox-group` used to import only the Lumo styles of the `checkbox` and didn't use to import the checkbox component itself. 

Fixed it, now when importing the group component, it provides the checkbox component as well.